### DISCUSSION TITLE
v0.6.0-M2

### DISCUSSION THREAD

**Discussion by erikerlandson** [erikerlandson/coulomb#294] [General]
- update spire dep to 0.18.0
- now with scala native build
- Thanks to @armanbilge and @cquiroz !

<hr /><em>This discussion was created from the release <a href='https://github.com/erikerlandson/coulomb/releases/tag/v0.6.0-M2'>v0.6.0-M2</a>.</em>

**Comment by erikerlandson** [comment]
@armanbilge  `v0.6.0-M2` should have your latest PR for spire and scala native

**Reply by armanbilge** [reply]
Thank you so much! We updated and this solved the binary-compatibility eviction problems we were having :)
